addons/hr_work_entry_contract/models/hr_contract.py

```diff
@@ -10,10 +10,11 @@
 from dateutil.relativedelta import relativedelta
 
 from odoo import api, fields, models, _
-from odoo.addons.resource.models.utils import string_to_datetime, Intervals
+from odoo.addons.resource.models.utils import string_to_datetime
 from odoo.osv import expression
 from odoo.tools import ormcache
 from odoo.exceptions import UserError
+from odoo.addons.hr_work_entry_contract.models.hr_work_intervals import WorkIntervals
```

Comment:
```suggestion

from .hr_work_intervals import WorkIntervals
from odoo.addons.resource.models.utils import string_to_datetime
```

You can shorten the import path if it's in the same directory, plus imports from odoo modules should be grouped separately (see [docs](https://www.odoo.com/documentation/17.0/contributing/development/coding_guidelines.html#imports))

Reply:
:+1: